de.metas.fresh/de.metas.fresh.base/src/main/jasperreports/de/metas/reports/bpartner_changes/report.jrxml

```diff
@@ -93,7 +93,7 @@ order By bpc.bpValue, bpc.tablename;]]>
 				<textElement>
 					<font size="14"/>
 				</textElement>
-				<text><![CDATA[Revisionsstelle Geschäftspartner]]></text>
+				<text><![CDATA[Stammdaten Mutationsprotokoll]]></text>
 			</staticText>
 			<textField pattern="dd.MM.yyyy">
 				<reportElement uuid="0915ffcd-95d0-495a-a086-48b740900673" x="41" y="20" width="50" height="12"/>
```

Comment:
So, this is legacy, and we need to correct it.
Please create resource files for English (en_US, en_GB) and German( de_CH, de_DE) and set that name in there.
Then use the resource in the report. 
Please, find an appropriate English name,